Javascript/q43.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q43</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .flex {
+      display: flex;
+      gap: 8px;
+      flex-wrap: wrap;
+      width: 300px;
+    }
+
+    .box {
+      width: 200px;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
+      const min = Number(document.querySelector("#demo0").value);
+      const max = Number(document.querySelector("#demo1").value);
+      const count = Number(document.querySelector("#demo2").value);
+      const arr = [];
+  
+      while(arr.length < count) {
+        arr.push(Math.round(Math.random() * (max - min + 1) + min ));
```

Comment:
I think this formula will give you random numbers between `min` and `max + 1` inclusive. The question is asking for a random number between `min` and `max` inclusive